apps/web/src/components/workflow/preview/ErrorPrettyRender.tsx

```diff
@@ -1,13 +1,20 @@
-import { css } from '@novu/novui/css';
-import { Button, Text, Title } from '@novu/novui';
-import { Center, Stack, VStack } from '@novu/novui/jsx';
-import { IconErrorOutline, IconExpandLess, IconExpandMore } from '@novu/novui/icons';
 import { useDisclosure } from '@mantine/hooks';
+import { css } from '@novu/novui/css';
+import { Text } from '@novu/novui';
 import { hstack } from '@novu/novui/patterns';
+import { Center, Stack } from '@novu/novui/jsx';
+import { IconErrorOutline, IconExpandLess, IconExpandMore } from '@novu/novui/icons';
 
 export function ErrorPrettyRender({ error: unparsedError }) {
   const [isExpanded, { toggle }] = useDisclosure();
   const error = 'response' in unparsedError ? unparsedError?.response?.data : unparsedError;
+  // TODO: error framework types enum
+  const isInvalidControlSyntax = error?.code === 'CompilingStepControlInvalidError';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add `@novu/framework` as a dependency of `@novu/web` and import the Error enum‏

Reply:
Yes, thats why I wasn't sure if I should. Maybe we could move some types to the shared lib? For this and for future things of this sort?

Reply:
I was mistaken, we already have `@novu/framework` as a dependency of `@novu/web`.

We already have a number of places that benefits from use Framework typings, primarily the discovery types. We will continue to need such typings so it's not unreasonable to have this dependency in place. Framework defines the source of truth for such values, there's no need to extract to shared lib.